Xamarin.Forms.Core/RefreshView.cs

```diff
@@ -20,7 +22,41 @@ public RefreshView()
 		}
 
 		public static readonly BindableProperty IsRefreshingProperty =
-			BindableProperty.Create(nameof(IsRefreshing), typeof(bool), typeof(RefreshView), false, BindingMode.TwoWay);
+			BindableProperty.Create(nameof(IsRefreshing), typeof(bool), typeof(RefreshView), false, BindingMode.TwoWay, coerceValue:OnCoerceIsRefreshing, propertyChanged: OnIsRefreshingPropertyChanged);
+
+		static void OnIsRefreshingPropertyChanged(BindableObject bindable, object oldValue, object newValue)
+		{
+			bool value = (bool)newValue;
+
+			if (value)
+			{
+				var resfrehView = ((RefreshView)bindable);
+				resfrehView.Refreshing?.Invoke(bindable, EventArgs.Empty);
+				if(resfrehView.Command != null)
+					resfrehView.Command.Execute(resfrehView.CommandParameter);
```

Comment:
so not checking if the command can execute before executing it then?

Reply:
No. My thinking here is that if IsRefreshing transitions to true for any reason then I want these events to all fire.  If these fire when they shouldn't then I would say that's an upstream bug.